audiofile/core/utils.py

```diff
@@ -62,18 +62,18 @@ def run(shell_command):
 def run_ffmpeg(infile, outfile, offset, duration):
     """Convert audio file to WAV file."""
     if duration:
-        cmd = f'ffmpeg -ss {offset} -i "{infile}" -t {duration} "{outfile}"'
+        cmd = ['ffmpeg', '-ss', str(offset), '-i', infile, '-t', str(duration), outfile]
     else:
-        cmd = f'ffmpeg -ss {offset} -i "{infile}" "{outfile}"'
+        cmd = ['ffmpeg', '-ss', str(offset), '-i', infile, outfile]
     run(cmd)
 
 
 def run_sox(infile, outfile, offset, duration):
     """Convert audio file to WAV file."""
     if duration:
-        cmd = f'sox "{infile}" "{outfile}" trim {offset} {duration}'
+        cmd = ['sox' infile, outfile, 'trim' str(offset) str(duration)]
     else:
-        cmd = f'sox "{infile}" "{outfile}" trim {offset}'
+        cmd = ['sox' infile, outfile, 'trim' str(offset)]
```

Comment:
And a few commas are missing here as well:

```suggestion
        cmd = ['sox', infile, outfile, 'trim', str(offset)]
```